docs/usage.md

```diff
@@ -49,256 +12,250 @@ It is recommended to limit the Nextflow Java virtual machines memory. We recomme
 
 ```bash
 NXF_OPTS='-Xms1g -Xmx4g'
-```
 
-## Running the pipeline
+## Samplesheet input
```

Comment:
is it more practical to require a samplesheet for the pipeline or provide the folder? IMO I am almost convinced having a samplesheet for this simple one overcomplicates things, but happy to be convinced by other users